src/backend/expungeservice/models/charge.py

```diff
@@ -14,127 +14,23 @@
 from expungeservice.models.charge_types.person_crime import PersonCrime
 from expungeservice.models.charge_types.schedule_1_p_c_s import Schedule1PCS
 from expungeservice.models.charge_types.unclassified_charge import UnclassifiedCharge
+from expungeservice.models.charge_classifier import ChargeClassifier
 
 
 class Charge:
-    classification = None
-
     @classmethod
     def create(cls, **kwargs):
-        cls.classification = None
         case = kwargs['case']
         statute = Charge.__strip_non_alphanumeric_chars(kwargs['statute'])
         level = kwargs['level']
         chapter = Charge._set_chapter(kwargs['statute'])
         section = Charge.__set_section(statute)
-        Charge._set_classification(case, statute, level, chapter, section)
+        cls.classification = ChargeClassifier(case, statute, level, chapter, section).classify()
```

Comment:
It seems this can also be changed to a staticmethod, and replace the class var with a local var?